python/semantic_kernel/connectors/ai/function_choice_behavior.py

```diff
@@ -73,7 +45,7 @@ class FunctionChoiceBehavior(KernelBaseModel):
         max_auto_invoke_attempts: The maximum number of auto invoke attempts.
         filters: Filters for the function choice behavior. Available options are: excluded_plugins,
             included_plugins, excluded_functions, or included_functions.
-        type: The type of function choice behavior.
+        type_: The type of function choice behavior.
```

Comment:
Why the `_` suffix? If it's to avoid collision with `type` maybe rename it to something more descriptive?

Reply:
Yeah it's the common way to denote a variable that overlaps with builtin types, @moon box 3 should we rename?

Reply:
Yeah, no worries from my part. It's handled internally anyway.